### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by icedebris** [johnaparker/fplanck#4]
Hello,

Thank you for writing a code to solve FPE. The examples seem to use some physics language, which I  find difficult to translate to "standard" math formulation. Can you please let me know how to setup a problem to solve the following problem. Suppose the two state-variables follow

dy_t = x_tdt
dx_t = f(x_t, y_t)dt + bdW_t,
where b is a constant and f(x_t, y_t) is a general function (it ensures that x_t and y_t have stationary dynamics on 0<x<inf, y<0<inf). 

I would like to solve for a stationary density g, which amount to solving the following FPE: -xg'_y - (f(x,y)g)'_x +b^2*g''_xx/2 = 0. Does you code allow to solve this problem? And if so, how one can do that?

Many thanks,
Igor

**Comment by JiejunHu** [comment]
> Hello,
> 
> Thank you for writing a code to solve FPE. The examples seem to use some physics language, which I find difficult to translate to "standard" math formulation. Can you please let me know how to setup a problem to solve the following problem. Suppose the two state-variables follow
> 
> dy_t = x_tdt dx_t = f(x_t, y_t)dt + bdW_t, where b is a constant and f(x_t, y_t) is a general function (it ensures that x_t and y_t have stationary dynamics on 0<x<inf, y<0<inf).
> 
> I would like to solve for a stationary density g, which amount to solving the following FPE: -xg'_y - (f(x,y)g)'_x +b^2*g''_xx/2 = 0. Does you code allow to solve this problem? And if so, how one can do that?
> 
> Many thanks, Igor

Dear Igor, is there an answer to your question? Thank you!

**Comment by icedebris** [comment]
No, I haven't gotten any reply. In the end, I simulated the process in Julia to obtain the density.

Best wishes,
Igor

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
